test/transactions/4_register_multisignature_account.js

```diff
@@ -13,357 +13,461 @@
+						passphrase,
+						secondPassphrase,
+						keysgroup,
+						lifetime,
+						minimum,
+					}),
+				).to.throw(
+					'Duplicated public key: 5d036a858ce89f844491762eb89e2bfbd50a4a0a0da658e4b2628b25b117ae09.',
+				);
 			});
+		});
+
+		describe('unsigned register multisignature account transaction', () => {
+			describe('when the register multisignature transaction is created without a passphrase', () => {
+				beforeEach(() => {
+					registerMultisignatureTransaction = registerMultisignatureAccount({
+						keysgroup,
+						lifetime,
+						minimum,
+					});
+					return Promise.resolve();
+				});
+
+				it('should have the type', () => {
+					return expect(registerMultisignatureTransaction)
+						.to.have.property('type')
+						.equal(transactionType);
+				});
+
+				it('should have the amount', () => {
+					return expect(registerMultisignatureTransaction)
+						.to.have.property('amount')
+						.equal(amount);
+				});
+
+				it('should have the fee', () => {
+					return expect(registerMultisignatureTransaction)
+						.to.have.property('fee')
+						.equal(fee);
+				});
+
+				it('should have the recipient id', () => {
+					return expect(registerMultisignatureTransaction)
+						.to.have.property('recipientId')
+						.equal('');
+				});
+
+				it('should have the sender public key', () => {
+					return expect(registerMultisignatureTransaction)
+						.to.have.property('senderPublicKey')
+						.equal(null);
+				});
+
+				it('should have the timestamp', () => {
+					return expect(registerMultisignatureTransaction).to.have.property(
+						'timestamp',
+					);
+				});
 
-			it('should have the recipient id', () => {
-				return expect(registerMultisignatureTransaction)
-					.to.have.property('recipientId')
-					.equal('');
+				it('should have the asset with the multisignature with the min, lifetime and keysgroup', () => {
+					return expect(registerMultisignatureTransaction)
+						.to.have.nested.property('asset.multisignature')
+						.with.all.keys('min', 'lifetime', 'keysgroup');
+				});
+
+				it('should not have the signature', () => {
+					return expect(registerMultisignatureTransaction).not.to.have.property(
+						'signature',
+					);
+				});
+
+				it('should not have the id', () => {
+					return expect(registerMultisignatureTransaction).not.to.have.property(
+						'id',
+					);
+				});
 			});
+		});
+	});
+	describe('#signMultisignature', () => {
+		describe('when invalid transaction is used', () => {
+			const transaction = {
+				amount: '10',
+				recipientId: '',
+				senderPublicKey:
+					'3358a1562f9babd523a768e700bb12ad58f230f84031055802dc0ea58cef1e1b',
+				timestamp: 59353522,
+				type: 0,
+				fee: '10000000',
+				recipientPublicKey: null,
+				asset: {},
+				signature:
+					'b84b95087c381ad25b5701096e2d9366ffd04037dcc941cd0747bfb0cf93111834a6c662f149018be4587e6fc4c9f5ba47aa5bbbd3dd836988f153aa8258e604',
+				id: '0000000000000000000',
+			};
+			const passphrase = '';
```

Comment:
I think a non-empty passphrase could make it clearer what's invalid about this situation. Preferably we would create a valid transaction for a plausible passphrase, and then modify it in some minimal way, e.g. change the last couple of bytes of the signature.